docs/development.md

```diff
@@ -346,7 +346,7 @@ You can optionally set the following variables:
 | `E2E_CONF_FILE`            | The path of the [E2E configuration file](https://cluster-api.sigs.k8s.io/developer/e2e.html#defining-an-e2e-config-file). | `${GOPATH}/src/sigs.k8s.io/cluster-api-provider-azure/test/e2e/config/azure-dev.yaml` |
 | `SKIP_CLEANUP`             | Set to `true` if you do not want the bootstrap and workload clusters to be cleaned up after running E2E tests.            | `false`                                                                               |
 | `SKIP_CREATE_MGMT_CLUSTER` | Skip management cluster creation.                                                                                         | `false`                                                                               |
-| `LOCAL_ONLY`               | Only use Kind local registry and run the subset of tests which don't require a remotely pushed controller image.          | `false`                                                                               |
+| `LOCAL_ONLY`               | Only use Kind local registry and run the subset of tests which don't require a remotely pushed controller image.          | `true`                                                                               |
 | `REGISTRY`                 | Registry to push the controller image. If `LOCAL_ONLY=true`, this is optional.                                            | `capzci.azurecr.io/ci-e2e`                                                            |
```

Comment:
" If `LOCAL_ONLY=true`, this is optional"

Looks like it's always optional since we're defaulting it

Reply:
good catch